agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml

```
@@ -65,6 +65,8 @@ spec:
       #  hostPath:
       #    path: /etc/os-release
       #    type: FileOrCreate
+      #- name: sys-tracing
+      #    path: /sys/kernel/debug
```

shouldn't this be:
```suggestion
      #- name: sys-tracing
      #  hostPath:
      #    path: /sys/kernel/debug
```
?